python-checks/src/test/java/org/sonar/python/checks/NotImplementedErrorInOperatorMethodsCheckTest.java

```diff
@@ -20,11 +20,25 @@
 package org.sonar.python.checks;
 
 import org.junit.Test;
+import org.sonar.python.checks.quickfix.PythonQuickFixVerifier;
 import org.sonar.python.checks.utils.PythonCheckVerifier;
 
 public class NotImplementedErrorInOperatorMethodsCheckTest {
   @Test
   public void test() {
     PythonCheckVerifier.verify("src/test/resources/checks/notImplementedErrorInOperatorMethods.py", new NotImplementedErrorInOperatorMethodsCheck());
   }
+
+  @Test
+  public void quickFixTest() {
+    var check = new NotImplementedErrorInOperatorMethodsCheck();
+    String codeWithIssue = "class MyClass:\n" +
+      "    def __lt__(self, other):\n" +
+      "        raise NotImplementedError()";
+    String codeFixed = "class MyClass:\n" +
+      "    def __lt__(self, other):\n" +
+      "        return NotImplemented";
+    PythonQuickFixVerifier.verify(check, codeWithIssue, codeFixed);
+    PythonQuickFixVerifier.verifyQuickFixMessages(check, codeWithIssue, NotImplementedErrorInOperatorMethodsCheck.QUICK_FIX_MESSAGE);
+  }
```

Comment:
Could you add another test documenting that something like:
```
def __mul__(self, other):
  ex = NotImplementedError()
  raise ex # Noncompliant
```
Would be quick fixed as:

```
def __mul__(self, other):
  ex = NotImplementedError()
  return NotImplemented
```

I think it's fine, but it's probably good to document that the quick fix will not take care of the now redundant variable assignment.

Reply:
done